Makefile

```diff
@@ -18,10 +18,10 @@ test-system: build
 
 build:
 	@mkdir -p dist
-	@go build -o dist ./cmd/...
+	@go build -ldflags='-s -X main.version=$(shell ./scripts/version.sh)' -o dist ./cmd/...
```

Comment:
How about `git describe --always --broken --dirty --tags` for this? I think it's more informative.

Reply:
Doesn't work

Reply:
J/k, idk the output is really weird `v0.0.1-123-g21fa9bc-dirty`